aws-py-static-website/__main__.py

```diff
@@ -0,0 +1,166 @@
+import json
+import mimetypes
+import os
+
+from pulumi import export, FileAsset, ResourceOptions, Config, Output
+import pulumi_aws
+import pulumi_aws.acm
+import pulumi_aws.cloudfront
+import pulumi_aws.config
+import pulumi_aws.route53
+import pulumi_aws.s3
+
+def get_domain_and_subdomain(domain):
+    parts = domain.split('.')
+    if len(parts) < 2:
+        raise Exception(f'No TLD found on ${domain}')
+    if len(parts) == 2:
+        return "", domain
+    subdomain = parts[0]
+    parts.pop(0)
+    return subdomain, '.'.join(parts) + '.'
+
+stack_config = Config('static-website')
```

Comment:
I've never used the Python SDK before, but you usually want to omit the argument to `pulumi.Config`. It will default to the project name, which is better since it means that things will probably do what you want if you ever rename the project by editing `Pulumi.yaml`. (Otherwise your existing stacks would still work, but any attempt to standup new stacks with the newer project name wouldn't ever find the configuration values.)